server/docker/postgresql/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3.5"
+
+services:
+  xef-vector-store-postgres:
```

Comment:
I like how you separated the databases here. We need tool space data and app-level data persisted. 👨‍🍳

Reply:
Thanks! Do you mean that we need to add another two databases?

Reply:
nah, I ment is great to have tow unrelated for vector store vs app data